### ISSUE TITLE
Uncaught Error: Cannot call function 'getPosition' on missing remote object 110

### ISSUE THREAD

**Issue by pixulainen** [atom/atom#20181]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.41.0 x64
**Electron**: 4.2.7
**OS**: Mac OS X 10.15.1
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Cannot call function 'getPosition' on missing remote object 110

```
At /Applications/Atom 2.app/Contents/Resources/electron.asar/renderer/api/remote.js:222

Error: Cannot call function 'getPosition' on missing remote object 110
    at throwRPCError (/Applications/Atom 2.app/Contents/Resources/electron.asar/browser/rpc-server.js:148:17)
    at /Applications/Atom 2.app/Contents/Resources/electron.asar/browser/rpc-server.js:406:5
    at EventEmitter.ipcMain.on.args (/Applications/Atom 2.app/Contents/Resources/electron.asar/browser/rpc-server.js:273:21)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (/Applications/Atom 2.app/Contents/Resources/electron.asar/browser/api/web-contents.js:368:21)
    at WebContents.emit (events.js:182:13)

```

### Commands

```
     -3:52.9.0 autocomplete-plus:confirm (input.hidden-input)
     -3:51.6.0 core:move-right (input.hidden-input)
  2x -3:23.1.0 core:backspace (input.hidden-input)
     -2:38.1.0 core:save (input.hidden-input)
     -1:56.9.0 emmet:insert-formatted-line-break-only (input.hidden-input)
     -1:56.9.0 editor:newline (input.hidden-input)
     -1:54.9.0 autocomplete-plus:confirm (input.hidden-input)
     -1:54.9.0 snippets:expand (atom-text-editor.editor.is-focused.autocomplete-active)
     -1:49.6.0 core:backspace (input.hidden-input)
     -1:43.6.0 core:move-right (input.hidden-input)
  6x -1:35.5.0 core:backspace (input.hidden-input)
     -1:27.2.0 emmet:insert-formatted-line-break-only (input.hidden-input)
     -1:27.2.0 editor:newline (input.hidden-input)
     -1:18.8.0 core:save (input.hidden-input)
     -0:17.9.0 emmet:insert-formatted-line-break-only (input.hidden-input)
     -0:17.9.0 editor:newline (input.hidden-input)
```

### Non-Core Packages

```
ask-stack 2.2.0 
atom-beautify 0.33.4 
atom-live-server 2.3.0 
busy-signal 2.0.1 
emmet 2.4.3 
file-icons 2.1.36 
highlight-selected 0.16.0 
intentions 1.1.5 
jshint 1.8.6 
linter 2.3.1 
linter-jscs 4.2.2 
linter-jshint 3.1.17 
linter-ui-default 1.8.0 
minimap 4.29.9 
minimap-pigments 0.2.2 
pigments 0.40.2 
teletype 0.13.4 
todo 0.6.0 
```

**Comment by Matt-Gleich** [comment]
Got this as well

**Comment by sadick254** [comment]
Thank you for opening this issue. This was issue present in electron versions 4 and below. The latest atom version `v1.51.0` uses electron version `5.0.3`. The issue was fixed on this commit https://github.com/electron/electron/commit/a8f2646ba682d65f94ea628ac11b1a5521bcedcd. The fix has been available from electron version `5.0.0`. I would advice you to upgrade to the latest atom version. I am closing this for now but happy to reopen if you still experience the issue.
